packages/components/src/internal/components/permissions/actions.ts

```diff
@@ -124,13 +127,117 @@ export function fetchContainerSecurityPolicy(
     });
 }
 
+export interface FetchedGroup {
+    id: number;
+    isProjectGroup: boolean;
+    name: string;
+    type: string;
+}
+export function fetchGroupPermissions(projectPath: string): Promise<FetchedGroup[]> {
```

Comment:
These methods should just be defined directly on the APIWrapper, the same way `fetchContainers` is. This will discourage anyone from depending on them directly.

Reply:
Cool! What is the utility of some of the functions, like `fetchContainerSecurityPolicy`, living in the actions.ts then?

Reply:
It also should only live on the APIWrapper, my guess is that it's only in actions.ts for historical reasons becasue it predates the addition of the security APIWrapper. If it's not a big burden I'd say move it over as part of this PR.